mnemosyne/cli.py

```
@@ -58,10 +58,26 @@ def _parse_int(value: str, name: str) -> int:
         _fail(f"{name} must be an integer: {value}")
 
 
+def _resolve_bank_name(bank_override: str | None = None) -> str:
+    """Resolve the selected memory bank from an override or MNEMOSYNE_BANK."""
+    value = bank_override if bank_override is not None else os.environ.get("MNEMOSYNE_BANK", "")
+    return value.strip() or "default"
+
+
 def _get_memory():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_get_memory`

(ANN202)